upgrading/downgrade.adoc

```diff
@@ -89,14 +83,7 @@ The **-excluder* packages add entries to the exclude directive in the host’s
 remove the following packages:
 +
 ----
-# yum remove atomic-openshift \
-    atomic-openshift-clients \
-    atomic-openshift-node \
-    atomic-openshift-master \
-    atomic-openshift-sdn-ovs \
-    atomic-openshift-excluder \
-    atomic-openshift-docker-excluder \
-    atomic-openshift-hyperkube
+# yum remove atomic-openshift-node \
 ----
```

Comment:
All high version pkgs need to be removed:
atomic-openshift-excluder-3.11.16-1.git.0.b48b8f8.el7.noarch
atomic-openshift-hyperkube-3.11.16-1.git.0.b48b8f8.el7.x86_64
atomic-openshift-node-3.11.16-1.git.0.b48b8f8.el7.x86_64
atomic-openshift-3.11.16-1.git.0.b48b8f8.el7.x86_64
atomic-openshift-docker-excluder-3.11.16-1.git.0.b48b8f8.el7.noarch
atomic-openshift-clients-3.11.16-1.git.0.b48b8f8.el7.x86_64

Reply:
Is it ok to only list the package names, not the specific versions?

Reply:
> Is it ok to only list the package names, not the specific versions?

yes, sure. I just list a example for u. You can just list the name.

Reply:
LGTM now

Reply:
LGTM now